app/hotels/src/allHotels/AllHotelsSearchList.js

```diff
@@ -15,8 +16,8 @@ type Props = {|
 |};
 
 function AllHotelsSearchList(combinedProps: Props) {
-  const hotels = idx(combinedProps, _ => _.data.edges) || []; // TODO: render 404 page instead of nothing
```

Comment:
Haha. Fuck.

Reply:
😄  Maybe it's about time to add this ESLint rule: https://eslint.org/docs/2.0.0/rules/no-warning-comments

Reply:
TODOs are OK, it's not finished yet. I just could do it properly here. But for example, Jakub is still working on search and instead of empty callback it's better to add TODO: https://github.com/kiwicom/react-native-app/blob/7fe19451ae28b4578ab3f98c6dd9da1ee9ff56eb/app/hotels/src/allHotels/searchForm/Guests.js#L52

We just need to fix them or delete at the end...